### PR TITLE
refactor(gatsby-source-shopify): inline defaultqueries

### PR DESCRIPTION
**Pull request by muescha**
## Description

changes:
- inline `defaultQueries`

*note: best view the changes with `[x] hide whitespace changes`*

## Related Issues

- #23922 `feat(gatsby-source-shopify): accept custom queries`

### CODE REVIEW

File: packages/gatsby-source-shopify/src/__tests__/index.js [hunk 1]
```diff
@@ -15,12 +16,7 @@ jest.mock(`../create-client`, () => {
             const [queryName] = Object.entries(mockQueries).find(
               ([queryName, queryString]) => queryString === query
             )
```
Comment (muescha) [inline packages/gatsby-source-shopify/src/__tests__/index.js:18]: > Hack alert.

if the query is a generated string and not the string exact from the `defaultQueries`, then this gets false: `queryString === new String(query)`

i don't know if this is more or less hacky:
```suggestion
            const queryName = /query Get(\w*)/.exec(query)[1].toLowerCase()
```
*works with rename `shoppolicies.json` -> `policies.json`*

Comment (muescha) [inline packages/gatsby-source-shopify/src/__tests__/index.js:18]: > Hack alert.

if the query is a generated string and not the string exact from the `defaultQueries`, then this gets false: `queryString === new String(query)`

i don't know if this is more or less hacky:
```suggestion
            const queryName = /query Get(\w*)/.exec(query)[1].toLowerCase()
```
*works with rename `shoppolicies.json` -> `policies.json`*

**Comment by muescha** [comment]
why the test **not** fail on macOS with filename `shopPolicies.json` 🤔 30bb1e4

**Comment by LekoArts** [comment]
I'll close this as soon a new major will be released with https://github.com/gatsbyjs/gatsby-source-shopify-experimental - so we won't be doing any work on the old plugin anymore (except security fixes)

**Pull request closed without merging**